### DISCUSSION TITLE
Low latency?

### DISCUSSION THREAD

**Discussion by Kieranwest** [datarhei/restreamer#828] [Q&A]
Hi, I asked a couple years back in #373 regarding low latency. Was this ever introduced? Loved using restreamer but had to switch due to the 7-10 second delay on streams. OvenMediaEngine can currently support sub-second streaming and it's the only thing keeping me from switching!

**Comment by ehspill** [comment]
same issue here regarding the webplayer, no way to remove buffering, highly annoying (5-6sec delay) (rtmp/rtsp recast has no such delay, so its not an encoder issue)
i also use oven media engine but they cant handle h265 via rtsp.
any update or help for adjusting the player to cut down the delay would be very helpful
